src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,40 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Arrays;
+
 public class FileWork {
+    public static final char GIVEN_CHARACTER = 'w';
+    public static final String WHITE_SPACE = " ";
+    public static final String EMPTY = "";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        File file = new File(fileName);
+        String newString = EMPTY;
+        try (BufferedReader reader = new BufferedReader(new FileReader(file))) {
+            String line = reader.readLine();
+            while (line != null) {
+                String[] words = line.split(WHITE_SPACE);
+                line = reader.readLine();
+                for (String word : words) {
+                    if (word.toLowerCase().charAt(0) == GIVEN_CHARACTER) {
+                        newString += word.toLowerCase().replaceAll("\\p{P}", EMPTY)
```

Comment:
you don't need to replace anything if you split by correct regex

Reply:
done.